### ISSUE TITLE
Overlapping feature notifications

### ISSUE THREAD

**Issue by johnbillion** [GoogleChromeLabs/wp-feature-policy#3]
## Issue Overview

When this plugin and [Reporting API](https://github.com/GoogleChromeLabs/wp-reporting-api) are activated together, the feature notifications overlap.

## Steps to Reproduce (for bugs)

1. Install and activate Feature Policy
2. Install and activate Reporting API

Observe that both plugins provide a feature notification and that they overlap.

## Screenshots / Video

<img width="499" alt="" src="https://user-images.githubusercontent.com/208434/54481713-51d42b00-4830-11e9-98a4-2ce847bfadde.png">

**Comment by felixarntz** [comment]
Good catch! I'd have to think how we can address this, since it's default core behavior, and the pointers are attached to their menu areas (which unfortunately are close together).

I'd argue it's not super critical, since you'll probably dismiss them after having seen them anyway, but I agree the initial experience is not that great.
